ansible/roles/nginx/vars/riak.yml

```diff
@@ -0,0 +1,21 @@
+---
+nginx_sites:
+- server:
+   file_name: "{{ deploy_env }}_riak"
+   listen: "{{ riak_port }}"
+   use_balancer: riakcs
+   upstream_port: "{{ riak_port }}"
+   proxy_set_headers:
+   - "Host $http_host"
+   - "X-Real-IP $remote_addr"
+   location1:
+      name: /
+      use_balancer: riakcs
+      proxy_redirect: ' off'
+      proxy_connect_timeout: 90
+      proxy_send_timeout: 90
+      proxy_read_timeout: 90
+      proxy_buffer_size: 64k
+      proxy_buffers: 8 64k
+      client_max_body_size: 10m
+      client_body_buffer_size: 128k
```

Comment:
this is used by the `site.j2` template. it loops through all the key vals and puts them in the location directive. (ctrl-f won't find a reference to it)